src/styles/style.scss

```diff
@@ -416,6 +389,29 @@ button {
   font-size: 14px;
   letter-spacing: 1px;
 }
+button.start-disabled {
+  border: 0;
+  padding: 10px;
+  color: white;
+  outline: none;
+  cursor:auto;
+  border-radius: 5px;
+  font-size: 14px;
+  letter-spacing: 1px;
+}
+button.start {
+  border: 0;
+  padding: 10px;
+  background-color: $selected-path-color;
+  color: white;
+  cursor: pointer;
+  outline: none;
+  border-radius: 5px;
+  font-size: 14px;
+  letter-spacing: 1px;
+  width: 150px;
+}
+
```

Comment:
We got an `atom/button` already. Try to reuse the class `button` and remove redundant code.
In addition I would always give the button the class `start` with basic styling and in the case we need to disable it, add the class `disabled`